Purchases.xcodeproj/project.pbxproj

```diff
@@ -17,7 +17,6 @@
 		2D22679125F2D9AD00E6950C /* PurchasesTests-Bridging-Header.h in Headers */ = {isa = PBXBuildFile; fileRef = 35262A291F7D783F00C04F2C /* PurchasesTests-Bridging-Header.h */; };
 		2D390255259CF46000DB19C0 /* MockPaymentDiscount.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2D390254259CF46000DB19C0 /* MockPaymentDiscount.swift */; };
 		2D3E29CE25E8009000456FA8 /* MockAttributionTypeFactory.swift in Sources */ = {isa = PBXBuildFile; fileRef = 37E35DA3E083FE37DAF15954 /* MockAttributionTypeFactory.swift */; };
-		2D4C02E426697E490038F877 /* RCLogLevel.h in Headers */ = {isa = PBXBuildFile; fileRef = 2D4C02E326697E490038F877 /* RCLogLevel.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
It's probably best to take a look at the files deleted/files added for a sense of what's changed. I don't pay much attention to the changes in the project file since they are often red herrings. I find moving anything, or adding anything can set off a chain of changes that is completely inconsequential. If there's something specific about the project file that doesn't make sense, I'm happy to investigate, though!

Reply:
That being said, if I update build settings, I'll try to point that out better